storage/rocksdb/rdb_datadic.cc

```diff
@@ -954,7 +955,7 @@ uint Rdb_key_def::get_primary_key_tuple(const Rdb_key_def &pk_descr,
   assert(m_pk_key_parts);
 
   /* Put the PK number */
-  rdb_netbuf_store_index(buf, pk_descr.m_index_number);
```

Comment:
nit: could you also change m_index_number as private field?

Reply:
It seems it is already a private field